toxav/msi.c

```diff
@@ -413,10 +466,49 @@ static uint8_t *msg_parse_header_out(MSIHeaderID id, uint8_t *dest, const void *
 
     return dest + value_len; /* Set to next position ready to be written */
 }
-static int send_message(Messenger *m, uint32_t friend_number, const MSIMessage *msg)
+
+/* Send an msi packet.
+ *
+ *  return 1 on success
+ *  return 0 on failure
+ */
+static int m_msi_packet(Tox *tox, int32_t friendnumber, const uint8_t *data, uint16_t length)
+{
+    // TODO(Zoff): make this better later! -------------------
+    size_t length_new = length + 1;
+    uint8_t *data_new = calloc(1, length_new);
```

Comment:
Please fix errors you see on Travis.

Reply:
@iphydf travis does not show me what the problem is:

```
Running TokTok style checker
Processing ../c-toxcore/toxav/audio.c
Processing ../c-toxcore/toxav/audio.h
Processing ../c-toxcore/toxav/bwcontroller.c
Processing ../c-toxcore/toxav/bwcontroller.h
Processing ../c-toxcore/toxav/groupav.c
Processing ../c-toxcore/toxav/groupav.h
Processing ../c-toxcore/toxav/msi.c
check-cimple: user error (Parse error near token: L (AlexPn 13195 487 34) IdVar "error")
The command ".travis/$JOB script" exited with 1.
```

how can i see the actual errors and where they occur?

Reply:
It's on line 487, column 34.

Reply:
The error is: you're using ALL_CAPS type names. You should be using Camel_Case names.

Reply:
@iphydf i understand the error , i just did not know what ```AlexPn 13195 487 34``` means.
now i see it linenumber and column.

Reply:
fix the first one. lets see if travis shows more